src/main/java/se/bjurr/violations/lib/example/OtherClass.java

```diff
@@ -4,11 +4,14 @@
  * No ending dot
  */
 public class OtherClass {
- public static String CoNstANT = "yes";
+ public static String CoNstANT = "yes"; 
```

Comment:
**Reporter**: FINDBUGS
**Rule**: MS_SHOULD_BE_FINAL
**Severity**: INFO
**File**: src/main/java/se/bjurr/violations/lib/example/OtherClass.java L7
**Source**: se.bjurr.violations.lib.example.OtherClass

Field isn't final but should be


   <p>
This static field public but not final, and
could be changed by malicious code or
        by accident from another package.
        The field could be made final to avoid
        this vulnerability.</p>


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<-1939212604>*